src/commands/tags/TagFileSystem.ts

```diff
@@ -76,7 +76,10 @@ export class TagFileSystem extends AzExtTreeFileSystem<ITagsModel> {
     }
 
     public async writeFileImpl(context: IActionContext, model: ITagsModel, content: Uint8Array, originalUri: Uri): Promise<void> {
-        const text: string = content.toString();
+        // weird issue when in vscode.dev, the content Uint8Array has a giant byteOffset that causes it impossible to decode
```

Comment:
You may want to ask around about this, maybe there is a reason for the offset. At the very least it'd be nice to know why.

Reply:
Asked in https://github.com/microsoft/vscode-discussions/discussions/469